pythx/cli/main.py

```diff
@@ -88,23 +97,29 @@ def refresh(staging, config):
 @opts.staging_opt
 @opts.html_opt
 @opts.yaml_opt
-def openapi(staging, mode):
+@opts.debug_opt
+def openapi(staging, mode, debug):
     """Return the API's OpenAPI spec data in HTML or YAML format.
 
     :param staging: Boolean whether to use the MythX staging deployment
     :param mode: The format to return the OpenAPI spec in (HTML or YAML)
+    :param debug: Boolean whether to set logging to debug or error
     """
+    if debug: utils.debug_mode()
     c = Client(staging=staging)  # no auth required
     click.echo(c.openapi(mode).data)
 
 
 @cli.command(help="Print version information of PythX and the API")
 @opts.staging_opt
-def version(staging):
+@opt.debug_opt
```

Comment:
```suggestion
@opts.debug_opt
```

Reply:
Btw: This bug (preventing overall CLI execution) highlights that we *really* need CLI tests. :laughing: